rules-documentation/logsource-guides/windows/category/process_creation.md

```diff
@@ -0,0 +1,179 @@
+# category: process_creation
+
+ID: 2ff912e8-159f-4789-a2ef-761292b32a23
+
+## Content
+
+<details>
+    <summary>Expand</summary>
+
+- [category: process\_creation](#category-process_creation)
+  - [Content](#content)
+  - [Description](#description)
+  - [Event Source(s)](#event-sources)
+  - [Logging Setup](#logging-setup)
+    - [Microsoft Windows Security Auditing](#microsoft-windows-security-auditing)
+      - [Process Creation](#process-creation)
+      - [Include Command-Line In Process Creation Events](#include-command-line-in-process-creation-events)
+    - [Microsoft-Windows-Sysmon](#microsoft-windows-sysmon)
+      - [Process Creation](#process-creation-1)
+  - [Event Fields](#event-fields)
+    - [Provider: Microsoft Windows Security Auditing / EventID: 4688](#provider-microsoft-windows-security-auditing--eventid-4688)
+    - [Provider: Microsoft-Windows-Sysmon / EventID: 1](#provider-microsoft-windows-sysmon--eventid-1)
+
+</details>
+
+## Description
+
+This logsource guide describes how to enable the necessary logging to make use of SIGMA rules that leverage the `process_creation` category
+
+## Event Source(s)
+
+This section describes the event source(s) that are required to be collected in order to receive the events used by the `process_creation` category detection rules
+
+```yml
+Provider: Microsoft Windows Security Auditing
+GUID: {54849625-5478-4994-a5ba-3e3b0328c30d}
+Channel: Security
+EventID: 4688
+```
+
+```yml
+Provider: Microsoft-Windows-Sysmon
+GUID: {5770385f-c22a-43e0-bf4c-06f5698ffbd9}
+Channel: Microsoft-Windows-Sysmon/Operational
+EventID: 1
+```
+
+## Logging Setup
+
+This section described how to setup logging in your environment
+
+### Microsoft Windows Security Auditing
+
+#### Process Creation
+
+- Subcategory GUID: `{0CCE922B-69AE-11D9-BED3-505054503030}`
+- Provider: `Microsoft Windows Security Auditing`
+- Channel: `Security`
+- Event Volume: `High`
```

Comment:
```suggestion
- Event Volume: `Medium to High`
```
There are other types which I definitely would label high and would want to differentiate from.

Reply:
Please do so.